src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
```

Comment:
Although it's generally acceptable to style the `body` tag, it would be better to avoid using tag names for styling as per the checklist. It's recommended to create a class for the body and apply styles to that class. However, this is a minor issue and not a strict rule, especially for global styles.